javascript/node/selenium-webdriver/firefox/index.js

```diff
@@ -367,14 +373,9 @@ class Driver extends webdriver.WebDriver {
       let service = createGeckoDriverService(binary);
       serverUrl = service.start();
       onQuit = () => service.kill();
+      caps.set(Capability.PROFILE, profile.encode());
```

Comment:
SGTM

I tweaked things a bit to avoid encoding empty profiles. See d59ccc169a7d0a23763d042badadc54241d8056e